diff --git a/target/linux/ramips/mt7621/base-files/etc/init.d/usb_irq b/target/linux/ramips/mt7621/base-files/etc/init.d/usb_irq
new file mode 100755
index 00000000000000..e0ce4ad6b7356a
--- /dev/null
+++ b/target/linux/ramips/mt7621/base-files/etc/init.d/usb_irq
@@ -0,0 +1,20 @@
+#!/bin/sh /etc/rc.common
+# Copyright (C) 2006 OpenWrt.org
+
+START=35
+
+start() {
+	NP=$(grep -c ^processor /proc/cpuinfo )
+	if [ "$NP" -gt 1 ]; then
+		INTERRUPT=$(ls /proc/irq/ | sed '/default/d')
+		USB_NUMBER=$(grep "xhci-hcd" /proc/interrupts | awk -F: '{print $1}' | sed 's/^ //')
+
+		for I in ${INTERRUPT}; do
+			if [[ "${I}" = "${USB_NUMBER}" ]]; then
+					echo "3" > /proc/irq/${I}/smp_affinity 2>/dev/null
+				else
+					echo "1" > /proc/irq/${I}/smp_affinity 2>/dev/null
+				fi
+		done
+	fi
+}
