diff --git a/target/linux/ramips/mt7621/target.mk b/target/linux/ramips/mt7621/target.mk
index 53b2f909aeb3b..476186cab083b 100644
--- a/target/linux/ramips/mt7621/target.mk
+++ b/target/linux/ramips/mt7621/target.mk
@@ -16,7 +16,7 @@ include $(INCLUDE_DIR)/target.mk
 # make Kernel/CopyImage use $LINUX_DIR/vmlinuz
 IMAGES_DIR:=../../..
 
-DEFAULT_PACKAGES += wpad-basic-mbedtls uboot-envtools
+DEFAULT_PACKAGES += wpad-openssl uboot-envtools
 
 define Target/Description
 	Build firmware images for Ralink MT7621 based boards.
