diff --git a/target/linux/ath79/generic/target.mk b/target/linux/ath79/generic/target.mk
index 53b2f909aeb3b..476186cab083b 100644
--- a/target/linux/ath79/generic/target.mk
+++ b/target/linux/ath79/generic/target.mk
@@ -0,6 +0,6 @@
 BOARDNAME:=Generic
 
-DEFAULT_PACKAGES += wpad-basic-mbedtls
+DEFAULT_PACKAGES += wpad-basic-openssl
 
 define Target/Description
 	Build firmware images for generic Atheros AR71xx/AR913x/AR934x based boards.
